### PR TITLE
stable/nginx-ingress repo is deprecated so updated it with the latest…

### PR DESCRIPTION
**Pull request by developer-guy**
… helm repository

### CODE REVIEW

File: content/en/docs/tutorials/acme/ingress.md [hunk 1]
```diff
@@ -65,13 +65,19 @@ For more information on securing your installation see: https://docs.helm.sh/usi
...
+```
+
 Update the helm repository with the latest charts:
 
 ```bash
 $ helm repo update
 Hang tight while we grab the latest from your chart repositories...
 ...Skip local chart repository
 ...Successfully got an update from the "stable" chart repository
+...Successfully got an update from the "stable" chart repository
```
Comment (meyskens) [inline content/en/docs/tutorials/acme/ingress.md:80]:
```suggestion
...Successfully got an update from the "ingress-nginx" chart repository
```

might be more logic

**Comment by meyskens** [comment]
Thanks for this! our tutorials really need a read through... just one not then LGTM

File: content/en/docs/tutorials/acme/ingress.md (hunk 1 above)
Comment (developer-guy) [reply content/en/docs/tutorials/acme/ingress.md:80]: good catch 🙏  thank you @meyskens, I have just done it.

**Pull request merged**